cfme/v2v/migration_plans.py

```diff
@@ -0,0 +1,598 @@
+
+    @property
+    def is_displayed(self):
+        return self.migration_request_details_list.is_displayed
+
+    def filter_by_vm_name(self, vm_name):
+        """Enter VM Name in search box and hit ENTER to filter the list of VMs.
+
+        Args:
+            vm_name(str): Takes VM Name as arg.
+        """
+        try:
+            self.filter_by_dropdown.item_select("VM Name")
+        except NoSuchElementException:
+            logger.info(
+                "filter_by_dropdown not present, "
+                "migration plan may not have started yet.Ignoring."
+            )
+        self.search_box.fill(vm_name)
+
+    def get_migration_status_by_vm_name(self, vm_name):
+        """Search VM using filter_by_name and return its status.
+
+        Args:
+            vm_name(str): Takes VM Name as arg.
+        """
+        try:
+            # Try to clear previously applied filters, if any.
+            self.clear_filters.click()
+        except NoSuchElementException:
+            # Ignore as button won't be visible if there were no filters applied.
+            logger.info("Clear Filters button not present, ignoring.")
+        self.filter_by_vm_name(vm_name)
+        status = {
+            "Message": self.migration_request_details_list.get_message_text(vm_name),
+            "Description": self.migration_request_details_list.get_progress_description(vm_name),
+            "Time Elapsed": self.migration_request_details_list.get_clock(vm_name),
+        }
+        return status
+
+    def filter_by_status(self, status):
+        """Set filter_by_dropdown to 'Status' and uses status arg by user to set status filter.
+
+        Args:
+            status(str): Takes status string as arg. Valid status options are:
+            ['Pending', 'Validating', 'Pre-migration', 'Migrating', 'VM Transformations Ccompleted',
+             'VM Transformations Failed']
+        """
+        try:
+            self.filter_by_dropdown.item_select("Status")
+            self.filter_by_status_dropdown.item_select(status)
+        except NoSuchElementException:
+            raise ItemNotFound(
+                "Migration plan is in Not Started State,"
+                " hence filter status dropdown not visible"
+            )
+
+    def sort_by(self, option):
+        """Sort VM list by using one of the 'Started','VM Name' or 'Status' option.
+
+        Args:
+            status(str): Takes status string as arg.
+        """
+        try:
+            self.sort_by_dropdown.item_select(option)
+        except NoSuchElementException:
+            raise ItemNotFound(
+                "Migration plan is in Not Started State," " hence sort_by dropdown not visible"
+            )
+
+    def plan_in_progress(self, vms_count=5):
+        """Reuturn True or False, depending on migration plan status.
+
+        If none of the VM migrations are in progress, return True.
+        """
+        if vms_count > 5:
+            self.items_on_page.item_select("15")
```

Comment:
@sshveta this is workaround as per me. As we have `v2vPaginator`; we can avoid this workaround. Let me know your thought on this.

Reply:
We can make it conditional based on `vm_count` , thoughts ? @sshveta

Reply:
@kedark3 : Pagination is only there for 5.10 . Even if vm_count is less or more than 5 , we can set the items per page to 15 and find the item in the list